sql/src/test/java/io/crate/lucene/ArrayLengthQueryTest.java

```diff
@@ -273,6 +274,10 @@ public void testArrayLengthEq0ReturnsNoElements() throws Exception {
     @Test
     public void testArrayLengthWithAllSupportedTypes() throws Exception {
         for (DataType<?> type : DataTypeTesting.ALL_TYPES_EXCEPT_ARRAYS) {
+            // This is temporary as long as interval is not fully implemented
+            if(type.equals(DataTypes.INTERVAL)) {
```

Comment:
Could also use `DataTypes.STORAGE_UNSUPPORTED` here, once we support interval storage the test would then reflect the change automatically (otherwise we'll forget about this "temporary" skipping)